Dockerfile

```diff
@@ -1,23 +1,28 @@
 FROM node:lts-alpine3.18 AS build-step
 ARG DYNAMIC_CONFIG=true
+ARG historyMode="history"
+ARG pathPrefix="/"
 
 WORKDIR /app
+RUN echo ${pathPrefix} > ./pathprefix
 COPY package*.json ./
 RUN npm install
 COPY . .
-RUN \[ "${DYNAMIC_CONFIG}" == "true" \] && sed -i 's/<!-- <script defer="defer" src=".\/config.js"><\/script> -->/<script defer="defer" src=".\/config.js"><\/script>/g' public/index.html
-RUN npm run build
+RUN \[ "${DYNAMIC_CONFIG}" == "true" \] && sed -i "s|<!-- <script defer=\"defer\" src=\"/config.js\"></script> -->|<script defer=\"defer\" src=\"${pathPrefix}config.js\"></script>|g" public/index.html
+RUN npm run build -- --historyMode="${historyMode}" --pathPrefix="${pathPrefix}"
 
 
 FROM nginx:1-alpine-slim
 
-COPY --from=build-step /app/dist /usr/share/nginx/html
```

Comment:
Not sure why bash and nano are installed here. The aim of the Dockerfile was to keep it as small as possible. I would understand why bash is installed, but nano does not seem to be used.

Reply:
My bad, forgot to remove this after "debugging".